lib/ash/query/function/ago.ex

```diff
@@ -3,28 +3,66 @@ defmodule Ash.Query.Function.Ago do
   Subtracts the given interval from the current time in UTC.
 
   For example:
-     deleted_at > ago(7, :days)
+     deleted_at > ago(7, :day)
```

Comment:
I think this example was incorrect - `:days` isn't handled base the original `case` expressions.